example/android/gradlew.bat

```diff
@@ -1,84 +1,84 @@
-@if "%DEBUG%" == "" @echo off

```

Comment:
the file of death

Reply:
yes, did the revert fix your issues?

Reply:
sadly, no. Maybe after it gets merged to develop, since it'll be the default branch when cloning